src/components/page/meta-page-info-flow.json

```diff
@@ -0,0 +1,17 @@
+{
+  "collectionName": "components_page_meta_page_info_flows",
```

Comment:
the title seems a bit off?
I think it should either be specific for its task (`flow-page-meta`?) or describing the intention (`reduced-meta`? `minimal-meta`?) 

I think I prefer version two